Docs for "zed manage" (i.e., compaction) (#4961)
brimdata/zed: docs/commands/zed.md

> Note that the branchlog meta-query source is not yet implemented.

### Manage
```
zed manage [options]
```

> Maintenance tasks are performed on a lake with the `manage` command.

Reviewer: Nit: Most of the other command sections start like this. I think it reads better too.
```suggestion
The `manage` command performs maintenance tasks on a lake.
```

Currently the only supported task is _compaction_, which reduces fragmentation

> by reading data objects in a pool and writing them back globally sorted and
> with their contents stored in contiguous large objects.

Reviewer: I don't think "globally sorted" and "contiguous" are accurate descriptions of what's going on here. How about "non-overlapping"?

```suggestion
by reading data objects in a pool and writing their contents back to large,
non-overlapping objects.
```

Author: @nwt: Thanks for this suggestion. Happy to take it. In case it wasn't noticed, the wording I was using was taken directly from text that I was removing elsewhere in this PR, i.e., what I'm quoting below was an earlier forward-looking statement that's still present right now in our docs but will be removed if this PR merges. Now that compaction has been implemented I certainly recognize our way of talking about it may have evolved, but that was not obvious to me.

> Note that data is easily compacted by reading from a fragmented pool and writing it back to a target pool so that it is globally sorted and compacted into contiguous large objects.

Reviewer: @philrz: I hadn't realized that.

If the `-monitor` option is specified and the lake is [located](#locating-the-lake)
via network connection, `zed manage` will run continuously and perform updates
as needed.  By default a check is performed once per minute to determine if
updates are necessary.  The `-interval` option may be used to specify an
alternate check frequency in [duration format](../formats/zson.md#23-primitive-values).

If `-monitor` is not specified, a single maintenance pass is performed on the
lake.

> The log output from `manage` provides a per-pool summary of the maintenance

Reviewer: Nit: I don't think "log" adds anything here.
```suggestion
The output from `manage` provides a per-pool summary of the maintenance
```
